lxd/storage_volumes_snapshot.go

```diff
@@ -640,6 +643,31 @@ func autoCreateCustomVolumeSnapshotsTask(d *Daemon) (task.Func, task.Schedule) {
 				continue
 			}
 
+			// Get list of cluster nodes
+			nodes, err := d.gateway.NodeStore().Get(d.gateway.Context())
+			if err != nil {
+				continue
+			}
+
+			// If there is more than one node (clustering), choose a random node for snapshotting.
```

Comment:
@monstermunchkin "choose a random node" is a little misleading, perhaps clarify that this is a 'stable' random node for this particular volume, so that it is chosen every time for this particular volume.

Also, as an aside, it would be interesting to test this logic against 3 node cluster with a few custom volumes and see how evenly distributed the hash algo is doing in terms of spreading the snapshots across the nodes.